src/pages/team.js

```diff
@@ -1,12 +1,208 @@
 import React from 'react'; // required for Gatsby
 import { AppLayout } from '../layouts/AppLayout';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import styles from "../styles/Box.css";
+
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+import Nishay from "../images/headshots/nsh.jpeg";
+import Ben from "../images/headshots/ben.jpeg";
+import Ivan from "../images/headshots/ivan.jpeg";
+import Shubh from "../images/headshots/shubh.jpeg";
+import Youssef from "../images/headshots/Youssef.jpeg";
+import Vasu from "../images/headshots/vasu.jpeg";
+import Nora from "../images/headshots/nora.jpeg";
+import Megan from "../images/headshots/meg.jpeg";
+import Alyson from "../images/headshots/ally.jpeg";
+import Jeremy from "../images/headshots/jeremy.jpeg";
+import David from "../images/headshots/david.jpeg";
+import Bernice from "../images/headshots/bernice.jpeg";
+import Luke from "../images/headshots/luke.jpeg";
+
+const cardInfo = [
+  {
+    description:
+      " Alyson is doing Master’s in Business Analytics (MSBA’22). She loves Music, food, and clean. Her favorite crypto is Cosmos",
+    title: "Alyson (Ran) Liu (Co-President)",
```

Comment:
Lets split all of these up into 'name' and 'position' and then in renderCard display the position in non-bold centered beneath the name